addons/web/static/src/model/relational_model/dynamic_list.js

```diff
@@ -38,6 +37,10 @@ export class DynamicList extends DataPoint {
         return this.config.domain;
     }
 
+    get evalContext() {
```

Comment:
not necessary to have a getter:
```js
this.evalContext = { ...this.context, companies: this.config.companies };
```
in the setup is equivalent (except that we won't get changes in the context, but I don't think it happens)